### DISCUSSION TITLE
Use monospace styling for proposed upgrade version in Renovate Ignore Notification

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#25332] [Suggest an Idea]
## Current situation

I just closed a PR that updated a dependency:

```diff
- mkdocs-material-extensions = "==1.2"
+ mkdocs-material-extensions = "==1.3"
```

- https://github.com/renovatebot/renovatebot.github.io/pull/332

Renovate then says:

> ### Renovate Ignore Notification
> 
> Because you closed this PR without merging, Renovate will ignore this update (==1.3). You will get a PR once a newer version is released. To ignore this dependency forever, add it to the `ignoreDeps` array of your Renovate config.
> 
> If you accidentally closed this PR, or if you changed your mind: rename this PR to get a fresh replacement PR.

## Wanted situation

I think Renovate should use `monospace` styling for the dependency version, like this:

> Because you closed this PR without merging, Renovate will ignore this update (`==1.3`). You will get a PR once a newer version is released. To ignore this dependency forever, add it to the `ignoreDeps` array of your Renovate config.

## Diff

```diff
- Because you closed this PR without merging, Renovate will ignore this update (==1.3).
+ Because you closed this PR without merging, Renovate will ignore this update (`==1.3`).
```

**Comment by viceice** [comment]
sounds good

**Reply by HonkingGoose** [reply]
Fixed by:

- #25333
